src/SFML/Graphics/Image.cpp

```diff
@@ -157,7 +158,15 @@ void Image::create(const Vector2u& size, const std::uint8_t* pixels)
 ////////////////////////////////////////////////////////////
 bool Image::loadFromFile(const std::filesystem::path& filename)
 {
-#ifndef SFML_SYSTEM_ANDROID
+#ifdef SFML_SYSTEM_ANDROID
+
+    if (priv::getActivityStatesPtr() != nullptr)
```

Comment:
What does it mean if this condition evaluates false? I'm not familiar with Android activity states.

Reply:
If there is an activity state, we are running as an activity, meaning the process was started using the activity manager. If we open up a console or just use adb to run a binary it will run natively without the activity manager, thus there will be no activity state in that case.